tests/unit/base/trained_model_serialization_test.cc

```diff
@@ -105,6 +109,30 @@ class TrainedModelSerializationFixture : public ::testing::Test
 		return save_success;
 	}
 
+	bool test_serialization(bool store_model_features = false)
```

Comment:
did you yet rebase against micheles merged patch? Surprised to still see this in here

Reply:
yes i did :) this only the part i want to add

Reply:
why do we need to add this? wasn't there something similar already?

Reply:
removes repetition of the same code this is a cleanup from the previous version :)